Feature/devopsdays 2019 (#581)
ordina-jworks/ordina-jworks.github.io: _posts/2019-11-28-DevOpsDays-2019.md

They can come from different perspectives. 
For a kid that just learned to tie shoelaces for example is an accomplishment because it is new to him/her. 
Although it means "nothing" to you because you've learned that a long time ago.

* **Learn how others make stuff and let others learn from you.** 
Other people no matter how professional they are, make mistakes as well and can learn things from you and you can learn from them.
    * Pair programming
    * Pair review
    * Celebrate your failure
    * Speak publicly

* **Find something your passionate about and keep tremendously interested in it.**
    * A hobby where you can screw things up without consequences

* .. etc

## Fight, Flight or Freeze – Releasing Organizational Trauma, by Matty Stratton
Imagine a Zebra grazing in the savanna of the African wilderness. 
Then suddenly a predator goes after the zebra. 
Of course, the zebra will run for his life (the flee state). 
When the zebra is caught by the predator, it 'freezes' because the zebra’s nervous system is overwhelmed, and it has no further solutions anymore. 
In the hope the predator will drop him and goes on. 

> After the zebra has managed to get away it literally shakes all his stress on and goes back to rest like nothing happened. 

Reviewer: ```suggestion
After the zebra has managed to get away it literally shakes all his stress off and goes back to rest like nothing happened. 
```

> Humans can react the same way zebras do, although we are not a zebra. 

Reviewer: True story. :-)

> Humans on the other hand can experience traumas long after a dreadful situation has occurred in the way on our human brain works.

Reviewer: ```suggestion
Humans on the other hand can experience traumas long after a dreadful situation has occurred because of the way our human brain works.
```

> Animals are not traumatized by routine threads to their live, where humans on the other hand, are really overwhelmed and often subject to traumatic symptoms of hyper arousal, shutdown and dysregulation in Peter Levine’s words. 

Reviewer: ```suggestion
Animals are not traumatized by routine threats to their live, where humans on the other hand, are really overwhelmed and often subject to traumatic symptoms of hyper arousal, shutdown and dysregulation in Peter Levine’s words. 
```

During day two, Matty Stratton, DevOps advocate at PagerDuty, was talking about crisis management and traumatic events within organizations and teams. 
Based on his own experience with post-traumatic stress. 
And gave good examples on how to better handle ‘traumatic’ events that can happen. 
But let’s start on how traumas can occur. 
Although it’s rather complex to explain.

* Traumas occurs when one's solutions (active response to threat) does not work. 
The case where your nervous system can’t handle it.

> Traumas can result from real or perceived threads. 

Reviewer: ```suggestion
Traumas can result from real or perceived threats. 
```

> A perception of a ‘threat’ can also cause of recall traumatic situations.

Reviewer: ```suggestion
A perception of a ‘threat’ can also cause or recall traumatic situations.
```

* Trauma is subjective and relative. 
Everyone experiences and handles stress in a different way. 
People from the military for example experience stress and traumas in a different way than people in software projects. 
But that doesn't mean that your experience isn’t real. 
Even though the experience from a soldier is much worse.

> But how does that apply to an Organization? 

Reviewer: ```suggestion
But how does that apply to an organization? 
```

You can deal with common situations and everything will go back to normal. 
Business as usual (the gray line in the normal range), but in the event of severe outages or a similar event, where the team’s capacity for a solution does not work and can lead to non-discharged stress (the red line). 

> To the organization.

Reviewer: Unnecessary sentence?

Hyperarousal (fight or flight) is being stuck on.

<img alt="Dunning-Kruger Effect" src="{{ '/img/2019-11-28-devopsdays-2019/Symptoms-of-undischarged-traumatic-stress.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">

Hyperarousal organizations are hyper-vigilant (like they are fighting Voldemort). 
They are hyper-aware of threats which causes a slowdown in moving forwarded. 
It’s not necessarily bad an organization to be vigilant. 
But rather when it loses a sane balance between vigilant and the will of moving forward and innovate.

> Organizations can also be in a "freeze" or Stuck off state. 

Reviewer: ```suggestion
Organizations can also be in a "freeze" or stuck off state. 
```

Organizations that believe that they are 'immune' to threats to stay in a standstill. 
So, they won't make any changes on how they operate. 

> It also happens that organizations fluctuating between the stuck on and off state.

Reviewer: ```suggestion
It also happens that organizations fluctuate between the stuck on and off state.
```

Humans like pattern recognition. 
We see signals that might remind us to events that happened before and activate us to respond the way we used to do last time. 
But the thing is that nothing is the same as it ever was back in the day. 
Because systems are becoming more complex as they grow.

Today’s challenge is to make organizations more resilient against dreadful events. 
Because resilient organizations are not traumatized by routine threats to their mission or business. 
Non-resilient organizations are readily overwhelmed and often subject to the symptoms of overreaction, shutdown and lack of regulated effort.

What you can do to be more resilient is to learn from your mistakes. 
Organize game days where you can practice an incident or outage. 
In a relax and safe way. 
It needs to be organized. 
They are not the same as stress tests.

## Conclusion

This is one of the conferences that has a different but interesting format. 

> Different to what I'm used to on conferences where I've been to so far. 

Reviewer: Changing it to "we" as the post has two authors.
```suggestion
Different to what we are used to on conferences where we have been to so far. 
```

You have on one hand talks during the morning and lightning talks and open spaces during the afternoon/evening where you can bring your own topics that you want to discuss with others. 

> What I also like is the fact that there’s a good verity between the amount of technical and soft-skills talks.

Reviewer: ```suggestion
What I also like is the fact that there’s a good variety between the amount of technical and soft-skills talks.
```

> That year the conference badges itself were actually seed mats that you can plant afterwards which I've done already. 

Reviewer: Weird sentence?
